e2e/cypress/integration/system_console/user_management_spec.js

```diff
@@ -0,0 +1,242 @@
+        });
+
+        // * User also receives email confirmation that email address has been changed.
+        checkResetEmail(testUser.username, testUser.email);
+
+        // # Logout, so that test user can login.
+        cy.apiLogout();
+
+        // * User cannot log in with old email address.
+        apiLogin(testUser.email, testUser.password).then((response) => {
+            expect(response.status).to.equal(401);
+
+            // # User logs in with username /password (no verification needed), then logs out again
+            cy.apiLogin({username: testUser.username, password: testUser.password}).apiLogout();
+
+            // # User logs in with new email address /password (no verification needed)
+            cy.apiLogin({username: newEmailAddr, password: testUser.password}).apiLogout();
+        });
+
+        // # Revert the changes.
+        cy.apiAdminLogin();
+        cy.visit('/admin_console/user_management/users').wait(TIMEOUTS.ONE_SEC);
+        resetUserEmail(newEmailAddr, testUser.email, '');
+    });
+
+    it('MM-T930 Users - Change a user\'s email address, with verification on', () => {
+        cy.visit('/admin_console/user_management/users').wait(TIMEOUTS.ONE_SEC);
+
+        // # Update Configs.
+        cy.apiUpdateConfig({
+            EmailSettings: {
+                RequireEmailVerification: true,
+            },
+        });
+
+        resetUserEmail(testUser.email, newEmailAddr, '');
+
+        cy.apiLogout();
+
+        // # Type email and password.
+        cy.get('#loginId').type(testUser.email);
+        cy.get('#loginPassword').type(testUser.password);
+        cy.findByText('Sign in').click();
+
+        // * Verify that logging in with the old e-mail works but requires e-mail verification.
+        cy.url().should('include', 'should_verify_email');
+
+        // # Log out.
+        cy.apiLogout();
+
+        // Verify e-mail.
+        verifyEmail(newUsername, newEmailAddr);
+        cy.wait(TIMEOUTS.HALF_SEC);
+
+        // * Verify that logging in with the old e-mail works.
+        cy.apiLogin({username: newEmailAddr, password: testUser.password}).apiLogout();
+
+        // # Revert the config.
+        cy.apiAdminLogin();
+        cy.apiUpdateConfig({
+            EmailSettings: {
+                RequireEmailVerification: false,
+            },
+        });
+    });
+
+    function resetUserEmail(oldEmail, newEmail, errorMsg) {
+        // # Search for the user.
+        cy.get('#searchUsers').clear().type(oldEmail).wait(TIMEOUTS.HALF_SEC);
+
+        // # Open the actions menu.
+        cy.get('[data-testid="userListRow"] .more-modal__right .more-modal__actions .MenuWrapper .text-right a').
+            click().wait(TIMEOUTS.HALF_SEC);
+
+        // # Click the Update email menu option.
+        cy.get('[data-testid="userListRow"] .more-modal__right .more-modal__actions .MenuWrapper .MenuWrapperAnimation-enter-done').
+            find('li').eq(4).click().wait(TIMEOUTS.HALF_SEC);
```

Comment:
I'm good as is but could be improved to make less susceptible to CSS/DOM structure change in the future.
```javascript
        cy.findByTestId('userListRow').within(() => {
            // # Open the actions menu.
            cy.findByText('Member').click().wait(TIMEOUTS.HALF_SEC);

            // # Click the Update email menu option.
            cy.findByLabelText('User Actions Menu').findByText('Update Email').click().wait(TIMEOUTS.HALF_SEC);
        });
```